src/Microsoft.PowerShell.Commands.Utility/commands/utility/XmlCommands.cs

```diff
@@ -359,6 +362,245 @@ protected override void StopProcessing()
         }
     }
 
+    #region ConvertTo-Clixml
+    /// <summary>
+    /// Implements ConvertTo-Clixml command.
+    /// </summary>
+    [Cmdlet(VerbsData.ConvertTo, "Clixml", SupportsPaging = true, DefaultParameterSetName = "ByPath", HelpUri = "")]
+    public sealed class ConvertToClicmlCommand : PSCmdlet, IDisposable
```

Comment:
```suggestion
    public sealed class ConvertToClixmlCommand : PSCmdlet, IDisposable
```

Reply:
@vexx32 I have made the correction. Thank you for pointing it out.